### PR TITLE
nginx ingress cloudflare origin cert

### PR DESCRIPTION
**Pull request by userbradley**
Adding docs on how to use cloudflare origin cert on nginx ingress

### CODE REVIEW

**Comment by TRafi9** [comment]
Id swap the first 2 sentences in the What section so you start with simple speak, as it gives context about what the page is solving, in simpler terms, at the beginning.
maybe add in some information for why we need both certificates/what purpose they both serve, as theres the root one and our origin cert

**Comment by userbradley** [comment]
> Id swap the first 2 sentences in the What section so you start with simple speak, as it gives context about what the page is solving, in simpler terms, at the beginning.

No, this is a technical site for technical people. if they dont understand ingress and SSl they wont grasp this. You could then argue that if they dont grasp it, they shoudlnt follow this, but this is just implimentation. no teaching how it works


> maybe add in some information for why we need both certificates/what purpose they both serve, as theres the root one and our origin cert

This I will add
